content/en/docs/deployment/device-provisioning-service.md

```diff
@@ -0,0 +1,242 @@
+
+### Client credential flow
+
+In this flow, the OAuth client is associated with the owner. The client must set the owner claim with the owner in the JWT access token.
+
+When using the `clientCredentials` flow, you cannot use `sub` because the owner will be the OAuth client. Therefore, you need to use another claim to identify the owner. We need to override default the hub configuration's section with  new `ownerClaim`, dps and web oauth clients:
+
+```yaml
+global:
+  ownerClaim: "https://plgd.dev/owner" # for mockoauthserver
+  oauth:
+    device:
+    - name: "plgd.dps"
+      clientID: my-client-id
+      clientSecret: my-client-secret
+      grantType: "clientCredentials"
+      scopes: ['openid']
+      audience: "https://api.example.com"
+```
+
+If you are using a mock OAuth2.0 server, add the following values to the `mockoauthserver` section:
+
+```yaml
+mockoauthserver:
+  oauth:
+  - name: "plgd.dps"
+    clientID: "test"
+    clientSecret: "test"
+    grantType: "clientCredentials"
+    scopes: ['openid']
+    audience: "https://api.example.com"
+    redirectURL: "https://mock.plgd.cloud/things"
+  - name: "plgd.web"
+    clientID: "test"
+    clientSecret: "test"
+    redirectURL: "https://mock.plgd.cloud/things"
+    scopes: ['openid']
+    useInUi: true
+```
+
+To allow download the Device Provisioning Service docker image by k8s, the following configuration needs to extend the configuration:
+
+```yaml
+deviceProvisioningService:
+  image:
+    dockerConfigSecret: |
+      {
+        "auths": {
+          "ghcr.io": {
+              "auth": "<DOCKER_AUTH_TOKEN>"
+          }
+        }
+      }
+```
+
+{{< note >}}
+
+To access ghcr.io, please reach out to us at [connect@plgd.dev](mailto:connect@plgd.dev) in order to request permission for your GitHub account to access the plgd device provisioning server images. You can refer to the [documentation](https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/) for instructions on how to allow access.
+
+{{< /note >}}
+
+## Configure Enrollment Groups
+
+The enrollment groups can be configured via deployment, utilizing the setup from the hub configuration to populate the values.
+
+{{< note >}}
+
+In the `deviceProvisioningService.enrollmentGroups[].attestationMechanism.x509.certificateChain` field, provide the certificate chain in PEM format that has used to sign the device manufacturer certificate (IDevId). The certificate chain should include the root CA certificate and the intermediate CA certificates.
```

Comment:
Is the order of certificates in the chain mentioned somewhere?

Reply:
fixed